Documentation/custom-scopes-claims-clients.md

```diff
@@ -22,6 +23,7 @@ Beyond the [required OpenID Connect claims][core-claims], and a handful of [stan
 | Name | Description |
 | ---- | ------------|
 | `groups` | A list of strings representing the groups a user is a member of. |
+| `id_provider` | The connector ID and the user ID assigned to the user at the provider. |
```

Comment:
Add an example somewhere that includes the full JSON object?

Reply:
Makes sense. Done!